pyhub/mcptools/google/sheets/tools/sheets_chunked.py

```
@@ -0,0 +1,270 @@
+                end_col_num //= 26
+
+        chunk_range = f"{sheet_name}!{start_col}{current_row}:{end_col}{end_row}"
+
+        try:
+            if batch_mode:
+                # Use batch update for better performance
+                result = await client.batch_update_values(spreadsheet_id, [(chunk_range, chunk_data)])
+                cells_in_chunk = sum(len(row) for row in chunk_data)
+            else:
+                # Use regular update
+                result = await client.update_values(spreadsheet_id, sheet_name, f"{start_col}{current_row}", chunk_data)
```

Pylint (3.3.7)

[error] Instance of 'GoogleSheetsAsyncClient' has no 'update_values' member

(E1101)